### ISSUE TITLE
Feature Request: Notifications when order expires or is (partially) filled

### ISSUE THREAD

**Issue by adamsmd** [etherdelta/etherdelta.github.io#232]
It would be useful to me if EtherDelta were to pop up a notification (as shown in [1]) when the status of an order changes such as when it expires (and I may thus want to reissue it) or is fully or partially filled (and I may thus want to do the trade I next planned after that one).

[1] https://developers.google.com/web/fundamentals/push-notifications/display-a-notification

**Comment by DeltaBalances** [comment]
As an experiment (before this issue) I made a script that does this. https://deltabalances.github.io/notification .
Not perfect but shows how it could work.

**Comment by MeoMix** [comment]
Yep! I agree that this would be a helpful feature. I'm not sure it's very high priority, though, and I suspect Zach is off working on FirstBlood or something else. :)
